Libraries/Hotcakes.PaypalWebServices/Hotcakes.PaypalWebServices.csproj

```diff
@@ -64,6 +64,11 @@
     <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.Bcl.AsyncInterfaces, Version=1.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51, processorArchitecture=MSIL">
+      <HintPath>..\..\packages\Microsoft.Bcl.AsyncInterfaces.1.1.0\lib\net461\Microsoft.Bcl.AsyncInterfaces.dll</HintPath>
```

Comment:
This appears to be a new dependency.  Was this required by Stripe or something else?

Reply:
Also, this is in PayPal.  Does it need to be in this PR?